prime-dotnet-webapi/openshift.dockerfile

```diff
@@ -52,13 +47,13 @@ COPY --from=build /opt/app-root/app/out/ /opt/app-root/app
 COPY --from=build /opt/app-root/app/Configuration/ /opt/app-root/app/Configuration/
 COPY --from=build /opt/app-root/app/entrypoint.sh /opt/app-root/app
 
-RUN apt-get update && \
-    apt-get install -yqq gpgv gnupg2 wget && \
+RUN yum update && \
```

Comment:
**NOTE:** Using RedHat requires we use `yum` instead of `apt-get` as the package manager.